pandas/tests/groupby/test_groupby.py

```diff
@@ -89,9 +89,12 @@ def max_value(group):
 
     applied = df.groupby("A").apply(max_value)
     result = applied.dtypes
+
+    # GH 28549
```

Comment:
a comment about something that is not there is not helpful if the reader doesn't know the context, which is pretty hard when just browsing code that is not part of the change itself. remove the comment.